code/script.js

```diff
@@ -0,0 +1,109 @@
+        `
+    })
+}
+getProfile()
+
+//Fetch for Technigo forked repos
+const getRepos = () => {
+    fetch (REPOS_URL)
+        .then(response => response.json())
+        .then(data => {
+            const forkedRepos = data.filter(repo => repo.fork && repo.name.startsWith('project-'))
+            forkedRepos.forEach((repo) => {
+                projectsContainer.innerHTML += `
+                <div class="single-project">
+                    <a href="${repo.html_url}" class="repo-name" target="_blank" style="text-transform: capitalize;">${repo.name}</a>
+                    <p style="text-transform: capitalize;">Branch: ${repo.default_branch}</p>
+                    <p class="last-push">Last push: ${repo.pushed_at.slice(0, 10)} - ${repo.pushed_at.slice(11, 16)}</p>
+                    <p class="commit-text" id="commit-${repo.name}">Commits: </p>
+                    <p class="comment-title"">Last comments: </p>
+                    <p class="comment-text" id="comment1-${repo.name}"> </p>
+                    <p class="comment-text" id="comment2-${repo.name}"> </p>
+                    <p class="comment-text" id="comment3-${repo.name}"> </p>
+                </div>
+                `
+        })
+        drawChart(forkedRepos.length)
+        fetchPulls(forkedRepos)
+        commitComments(forkedRepos)
+        })
+}
+
+//Fetch pull request
+const fetchPulls = (allRepositories) => {
+    allRepositories.forEach((repo) => {
+        const PULL_URL = `https://api.github.com/repos/Technigo/${repo.name}/pulls?per_page=100`
+        fetch(PULL_URL)
+        .then((response) => response.json())
+        .then((data) => {
+            const myPullRequest = data.find(pull => pull.user.login === repo.owner.login)
+            if (myPullRequest){
+                fetchCommits(myPullRequest.commits_url, repo.name)
+            } else {
+                document.getElementById(`commit-${repo.name}`).innerHTML = 'Commits: 0'
+            }
+        })
+    })
+}
+
+//Fetch nr of commits
+const fetchCommits = (myCommitsUrl, RepoName) => {
+    fetch(myCommitsUrl)
+    .then((response) => response.json())
+    .then((data) => {
+        document.getElementById(`commit-${RepoName}`).innerHTML += data.length
+    })
+    }
+
+//Fetch last commit messages
+const commitComments = (forkedRepos) => {
+    forkedRepos.forEach((repo) => {
+        const COM_URL = `https://api.github.com/repos/dandeloid/${repo.name}/commits`
+        fetch(COM_URL)
+        .then((response) => response.json())
+        .then((data) => {
+
+            if (data[0].author.login === 'dandeloid'){
+            const commitDate1 = new Date (data[0].commit.committer.date).toDateString()
+            document.getElementById(`comment1-${repo.name}`).innerHTML += `${commitDate1}: "${data[0].commit.message}"`
+            }
+            if (data[1].author.login === 'dandeloid'){
+            const commitDate2 = new Date (data[1].commit.committer.date).toDateString()
```

Comment:
Maybe this part could have an indentation to easily see that it is inside the if? 😄